src/utils/navigation.cljs

```diff
@@ -6,3 +6,11 @@
 (defn navigate-back
   []
   (rf/dispatch [:navigate-back]))
+
+(defn hide-bottom-sheet
+  []
+  (rf/dispatch [:hide-bottom-sheet]))
+
+(defn dismiss-keyboard
+  []
+  (rf/dispatch [:dismiss-keyboard]))
```

Comment:
this is not related to navigation right?

Reply:
both bottom sheet and keyboard

Reply:
hi @flexsurfer, Yes they are not related to navigation.

https://github.com/status-im/status-mobile/pull/20730#discussion_r1689673476

> If we feel like utils.navigation is not the best name we can change in a separate PR

We can rename and move this namespace to a single file, maybe utils.functions etc.? wdyt? (Otherwise we have to create so many utils files, for navigation, bottom sheet, keyboard)

Reply:
Agree with you guys, the original name `navigation` is too specific, but `utils.functions` is too generic because almost everything is a function in Clojure. Maybe `utils.events` or `utils.user-actions`? A namespace docstring would help a lot because it would allow us to set the purpose of the namespace because even `utils.events` can be confusing and a developer may think any kind of common event should be there, but in reality only the highly reusable dispatches should be there, otherwise the namespace could become a black hole of dispatches.

Reply:
created an issue, so we don't forget this
https://github.com/status-im/status-mobile/issues/20880